third_party/caffe/Dockerfile

```diff
@@ -29,7 +29,8 @@ WORKDIR $CAFFE_ROOT
 
 # FIXME: use ARG instead of ENV once DockerHub supports this
 # https://github.com/docker/hub-feedback/issues/460
-ENV CLONE_TAG=1.0
+# ENV CLONE_TAG=1.0
+ENV CLONE_TAG=master
```

Comment:
Sorry for the late review, we are busy these days, thanks for your contribution.
we can't switch the version to master for it's not a stable version, but no doubt your commit is meaningful. Maybe it's better to use 1.0 here. If someone uses the Clip operator, he can use the master version to do the validation.

Reply:
Ok, I reverted to tag 1.0 and add a note regarding CAFFE validation with Clip op